text_formatter_other.go

```diff
@@ -0,0 +1,3 @@
+// +build !js
```

Comment:
I'm not sure this file is useful, this build constraint should already be covered by terminal_check_not_appengine.go file.